Jenkinsfile

```diff
@@ -16,7 +16,7 @@ buildJavaProject(
         productionVisibility: 'docker.io')
 
 sconeBuildUnlocked(
-        nativeImage:     "nexus.iex.ec/$repositoryName:$buildInfo.shortCommit",
+        nativeImage:     "docker-regis.iex.ec/$repositoryName:$buildInfo.imageTag",
```

Comment:
What about extracting `docker-regis.iex.ec` to a var - such as `repositoryName`? It would be slightly easier to update if we ever need to do so.